mlir/test/Target/LLVMIR/arm-sme.mlir

```diff
@@ -334,3 +334,137 @@ llvm.func @arm_sme_vector_to_tile_vert(%tileslice : i32,
       (i32, i32, vector<[2]xi1>, vector<[2]xf64>) -> ()
   llvm.return
 }
+
+// -----
+
+llvm.func @prevent_dce.nxv16i8(vector<[16]xi8>)
```

Comment:
do we need to prevent DCE for intrinsics? We get away without it for all the other ones, I should think we do for these too?

Reply:
The other intrinsic mutate ZA, these just read a value out of it (no mutation), which assume would be seen as dead code.

Reply:
Looks like these can be removed anyway (I guess it does not try anything clever with intrinsics).